### ISSUE TITLE
I have Windows PowerShell and it can complete arguments starting by - my Windows version 20h2

### ISSUE THREAD

**Issue by crramirez** [NotTheDr01ds/WSLTabCompletion#1]

**Comment by NotTheDr01ds** [comment]
Can you provide more detail?

Yes, both Windows PowerShell and PowerShell Core complete arguments for PowerShell commands starting with `-`, but `wsl.exe` is not an internal PowerShell command (nor cmdlet).

**Comment by crramirez** [comment]
Hello,

I want just to delete the issue but you answered first. This is due to this paragraph in the Limitations:

"Windows PowerShell (the version installed with Windows by default) does not offer completions for arguments starting with -. This makes the feature fairly limited there. Since Windows PowerShell is end-of-life, Microsoft has no plans to fix this."

But if I write wsl and hit tab I can see all the parameters. But now I understood if I type wsl - and Tab I won't receive any completion. But for me, it works just fine. I found you in StackOverflow in the WSLg answer and decided to see if you were on GitHub and I found this interesting project.

Regards,
Carlos
